src/ApiClient.php

```diff
@@ -47,6 +45,11 @@ class ApiClient implements ApiClientInterface
      */
     protected $uriFactory;
 
+    /**
+     * @var Token
+     */
+    protected $token;
```

Comment:
Now I noticed we are not adding types to properties, but it will be forced by higher levels of PHPStan anyway.

Reply:
lets fix all issues reported after bumping up the level